crates/aptos/src/governance/mod.rs

```diff
@@ -183,3 +203,166 @@ impl std::fmt::Display for ProposalMetadata {
         )
     }
 }
+
+#[derive(Clone, Debug, Default, Serialize)]
+pub struct ScriptHash {
+    hash: String,
+    bytecode: String,
+}
+
+#[derive(Parser)]
+pub struct PrepareScript {
+    /// Path to the Move package that contains the execution script
+    #[clap(long, parse(from_os_str))]
+    pub path: PathBuf,
+}
+
+#[async_trait]
+impl CliCommand<ScriptHash> for PrepareScript {
+    fn command_name(&self) -> &'static str {
+        "PrepareScript"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<ScriptHash> {
+        let build_config = BuildConfig {
+            additional_named_addresses: BTreeMap::new(),
+            generate_abis: false,
+            generate_docs: false,
+            ..Default::default()
+        };
+
+        let compiled_package = compile_move(build_config, self.path.as_path())?;
+        let scripts_count = compiled_package.scripts().count();
+
+        if scripts_count != 1 {
+            return Err(CliError::UnexpectedError(format!(
+                "Prepare 1 script a time. Found {} scripts.",
```

Comment:
Could we be clearer here? Is this to do with the number of `.move` files?

Reply:
Sure. I can make this clearer. The reason is that we don't want to users to put many scripts under one package. It is going to be messy for them to track hashes and the scripts.